### ISSUE TITLE
Buzzer

### ISSUE THREAD

**Issue by frendhomeio** [diyruz/geiger#6]
Hello! Could you add buzzer support? (Signal - BUZ. Contact 10 (P1.5) Е18МS)

**Comment by mapoby** [comment]
Yes, feature is highly needed.

**Comment by Calimerorulez** [comment]
Yes, please add. I thought mine was broken, but just saw it isn't supported jet :)
